packages/firebase_ml_vision/android/src/main/java/io/flutter/plugins/firebasemlvision/TextRecognizer.java

```diff
@@ -125,4 +144,33 @@ private void addData(
 
     addTo.put("text", text);
   }
+
+  private FirebaseVisionCloudTextRecognizerOptions parseCloudOptions(
+      Map<String, Object> options, MethodChannel.Result result) {
+    FirebaseVisionCloudTextRecognizerOptions.Builder builder =
+        new FirebaseVisionCloudTextRecognizerOptions.Builder();
+
+    boolean enforceCertFingerprintMatch = (Boolean) options.get("enforceCertFingerprintMatch");
```

Comment:
This can result in an NPE if the key is missing int the map, we should probably fail more gracefully with a message saying the field was missing? (or have a default if there's a reasonable default value?)